src/pages/City/AddAdministratorModal/AddAdministratorModal.tsx

```diff
@@ -162,7 +166,7 @@ const AddAdministratorModal = (props: Props) => {
           <AutoComplete
             className="adminTypeSelect"
             options={[
-              { value: "Голова Станиці" },
+              { value: "Голова Станиці", disabled: activeUserRoles.includes("Заступник Голови Станиці") ? true : false },
```

Comment:
Wouldn't it be simplier to write disabled: activeUserRoles.includes ("Deputy Head of the Village") as "includes" returns true or false itself?

Reply:
Done